generators/sram-bist/src/main/scala/sram-harness/SramHarness.scala

```diff
@@ -44,6 +44,7 @@ class SramHarness(params: SramHarnessParams)(implicit p: Parameters)
   val gatedClock = if (p(WithChiseltestSramsKey).isDefined) {
     (clock.asBool & io.sramEn).asClock
   } else {
+    // (clock.asBool & io.sramEn).asClock
```

Comment:
Take this out, I think it might have gotten carried over by the merge conflicts.